apps/api/src/app/events/e2e/send-message-push.e2e.ts

```diff
@@ -39,7 +41,7 @@ describe('Trigger event - Send Push Notification - /v1/events/trigger (POST)', (
   });
 
   after(() => {
-    process.env.IS_MULTI_PROVIDER_CONFIGURATION_ENABLED = 'false';
+    process.env.IS_MULTI_PROVIDER_CONFIGURATION_ENABLED = ORIGINAL_IS_MULTI_PROVIDER_CONFIGURATION_ENABLED;
```

Comment:
To keep consistency on the default state of the system for this feature flag.

Reply:
yeah, that should help us do more consistent e2e on every test.
did you have the chance to see that after this change the flakiness faded?

Reply:
Yes, both locally and in the CI. Actually the checks passed at first try!